hack/generator/pkg/codegen/testdata/Flattening/FlattenedPropertiesAreFlattened.golden

```diff
@@ -0,0 +1,16 @@
+// Code generated by k8s-infra. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package v1alpha1api20200101
+
+//Generated from: https://test.test/schemas/2020-01-01/test.json
+type Test struct {
+	InnerFlat    *Inner `json:"inner_flat,omitempty"`
+	InnerNotFlat *Inner `json:"inner_not_flat,omitempty"`
```

Comment:
Note that this doesn’t do the desired thing yet, due to `gojsonschema` not exposing custom attributes.

Reply:
What needs to happen? Does this block merging?

Reply:
I _think_ @Porges is referring to our inability to write golden file tests with Swagger input, which means we can't actually _set_ the flattening flag.

Reply:
Added issue to track: https://github.com/Azure/azure-service-operator/pull/1548#discussion_r647856588